src/lib/components/Tools/DrawTool.svelte

```diff
@@ -4,13 +4,14 @@
 	import { toolState, canvasTouched, socket, user } from '$lib/stores/stateStore';
 	import { onMount } from 'svelte';
 	import { writable } from 'svelte/store';
-	import { ToolState } from '$lib/types';
+	import { ToolState, type ToolSuccess } from '$lib/types';
 
 	const STROKE_WIDTH = 7;
 	let currentCommandId = writable<number | null>(null);
 
 	onMount(() => {
-		$socket.on('startDrawSuccess', (data) => {
+		if ($socket === undefined) return;
```

Comment:
Is undefined a valid value that $socket can have?

Reply:
Dette er blevet refactored nu i en nu branch